lib/customer_io.dart

```diff
@@ -1,8 +1,18 @@
-
+import 'customer_io_config.dart';
 import 'customer_io_platform_interface.dart';
 
 class CustomerIo {
-  Future<String?> getPlatformVersion() {
-    return CustomerIoPlatform.instance.getPlatformVersion();
+  const CustomerIo._();
+
+  static CustomerIOPlatform get _customerIO => CustomerIOPlatform.instance;
+
+  static Future<String?> getPlatformVersion() {
+    return _customerIO.getPlatformVersion();
+  }
+
+  static Future<void> initialize({
```

Comment:
Ah, this must be why the other Dart file said `await CustomerIO.initialize()`. 

Does this function have to be a `Future` since the native SDK `initialize()` function isn't async?

Reply:
This doesn't have to be `Future ` but it's a common practice in plugin development that I have noticed, they return back with success/failures. 

I believe it's for the sake of state management as if they want to perform all initializations first and then move on to the other screens, native platforms provide it themselves, but maybe Cross platform doesn't